arakoon.opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "1.2"
+name: "arakoon"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+authors: ["Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+          "Jan Doms <jan.doms@openvstorage.com>"]
```

Comment:
not sure if anyone is still listening there :wink: 
(so feel free to remove me from that list)